packages/admin-ui-core/src/tokens/borderRadius.ts

```diff
@@ -1,6 +1,5 @@
 export const borderRadius = {
-  default: '4px',
-  flat: '0px',
-  pill: '100px',
-  circle: '100%',
+  'border-radius-none': '0rem',
+  'border-radius-base': '0.25rem',
+  'border-radius-pill': '100%',
```

Comment:
Just one thing that I didn't notice before: The tokens shouldn't have the `border-radius` prefix, right?